java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3390,4 +3390,27 @@ public void geopos(BaseClient client) {
                 assertThrows(ExecutionException.class, () -> client.geopos(key2, members).get());
         assertTrue(executionException.getCause() instanceof RequestException);
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void append(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String value1 = String.valueOf(UUID.randomUUID());
+        String key2 = UUID.randomUUID().toString();
+        String key3 = UUID.randomUUID().toString();
+
+        assertEquals(value1.length(), client.append(key1, value1).get());
+        assertEquals(value1.length() * 2L, client.append(key1, value1).get());
+        assertEquals(value1 + value1, client.get(key1).get());
+
+        // Non Existing Key
+        assertEquals(1L, client.append(key2, "b").get());
```

Comment:
we can remove this test.  Line 3403 does the same thing.

Reply:
It's for this line https://github.com/Bit-Quill/glide-for-redis/pull/281/files#r1605442233. `key3` can be removed as `key2` can be used for the test for wrong type.